js/index.js

```diff
@@ -42,8 +42,40 @@ window.addEventListener('load', event => {
   // Bind the click event of each element to a function
   document.querySelectorAll('.card').forEach(card => {
     card.addEventListener('click', () => {
-      // TODO: write some code here
-      console.log(`Card clicked: ${card}`);
+      let cardName = card.getAttribute("data-card-name");
+      console.log(`card picked is ${cardName}`);
+      card.querySelectorAll('.card div').forEach(div =>{
+        div.classList.toggle("front")
+        div.classList.toggle("back")
+      })
+      console.log(`bla`,memoryGame.pickedCards)
+      memoryGame.pickedCards.push(cardName)
```

Comment:
You should push the entire element here, that way you would have a point of reference later when you need to do DOM manipulation(you can still store the card name in a separate variable)
```
memoryGame.pickedCards.push(card)